samples/javascript_nodejs/13.core-bot/package.json

```diff
@@ -19,7 +19,6 @@
         "@microsoft/recognizers-text-data-types-timex-expression": "^1.1.4",
         "botbuilder": "^4.5.0-preview.69412",
```

Comment:
@stevengum, do we need an explicit ref to botbuilder? or is it implicit in botbuilder-dialogs, ai and testing?

Reply:
Good ask; we do need the explicit ref to `botbuilder`. If this dependency is referenced in the other libraries it isn't exposed in the same way that `botframework-schema` and `botbuilder-core` is exposed through `botbuilder`.

Reply:
k, thanks